QgisModelBaker/libqgsprojectgen/generator/generator.py

```diff
@@ -61,6 +61,8 @@ def __init__(self, tool, uri, inheritance, schema=None, pg_estimated_metadata=Fa
         self._db_connector.stdout.connect(self.print_info)
         self._db_connector.new_message.connect(self.append_print_message)
 
+        self._ignored_layers = []  # List of layers to ignore set by 3rd parties
```

Comment:
```suggestion
        self._additional_ignored_layers = []  # List of layers to ignore set by 3rd parties
```
Because otherwise I would assume these are the ignored layers that `get_ignored_layers` return.